packages/tasks/src/hardware.ts

```diff
@@ -84,6 +84,10 @@ export const SKUS = {
 				tflops: 37.42,
 				memory: [48],
 			},
+			A16: { 
+				tflops: 17.972, // source: https://www.techpowerup.com/gpu-specs/a16-pcie.c3794
+				memory: [64], // this one you do not have to add, as its 4x.
```

Comment:
Yeah, it's a bit confusing as this is a bundle of 4 GPUs in a single card. I'm not sure if we should report the per-GPU specs instead, as the use cases of 4x16 GB are different to 1x64 GB cards.

Reply:
Alright I will just remove it since it will be inaccurate anyways, since its basically 1 gpu, but 4 parts? I am always confused on those.

Reply:
Yes, another idea would be to use the per-unit specs: 4.49 tflops and 16 GB of RAM, but that could be somewhat confusing as well.